src/server/resolvers.ts

```diff
@@ -178,6 +187,17 @@ export const resolvers: Resolvers = {
 		note: async question => (await question).note || null,
 		prompt: async question => (await question).prompt,
 	},
+	Event: {
+		end: async event => (await event).end.getTime(),
+		id: async event => (await event)._id.toHexString(),
+		location: async event => (await event).location,
+		start: async event => (await event).start.getTime(),
+		summary: async event => (await event).summary,
+		type: async event => {
```

Comment:
I'm kinda surprised that typescript allows type as a name seeing it's a reserved keyword

Reply:
lol yeah we should make this `eventType`